roles/openshift_health_checker/library/aos_version.py

```diff
@@ -26,6 +27,13 @@
 except ImportError as err:
     IMPORT_EXCEPTION = err
 
+# NOTE: yum does not work with Python 3, but since the openshift-ansible code
+# base is tested against both Python 2 and Python 3, we write the code below for
+# compatibility with the latter:
+PY3 = sys.version_info[0] == 3
+if PY3:
+    basestring = str
```

Comment:
Well, that approach causes a `pylint` error because it looks at code statically and thinks that 1) under py2 the builtin name `basestring` is being overwritten; and 2) under py3 the name `basestring` is invalid for a constant. Instead, we can import `six` from `ansible.module_utils`.